cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -19,11 +19,6 @@
 # managed result is of that. Start everything below that directory.
 KUBECTL=/usr/local/bin/kubectl
 
-if [ -z "$APISERVER_URL" ] ; then
```

Comment:
Same justification. APISERVER_URL is just replaced with a single hardcoded value below.